apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md

```diff
@@ -0,0 +1,280 @@
+        
+    DELETE FROM public.inpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.outpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.snf_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    
+    DELETE FROM public.partd_events
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiary_monthly
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries_history
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';`
+
+
+## Load Synthetic Data in Hosted Environments
+
+1. Before loading data, we need to make sure in TEST, SBX or PROD there are no collisions for the main properties in synthetic data i.e. bene_id, mbi_num, claim_id, etc. Each of the queries should return zero records. Aggregate RIF file data from all claim types, beneficiaries, and Part D Events located in `apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-synthea/` to generate the following queries for the various synthea end-state properties: 
+
+        -- For Beneficiary
+        Select count(*) from public.beneficiaries where mbi_num in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where bene_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where hicn_unhashed in (‘XXX’,’YYY’,…);
+
+        
+        -- For PDE
+        Select count(*) from public.partd_events where pde_id in (‘XXX’,’YYY’,…);
+
+
+        -- For all claim types
+
+        Select count(*) from public.carrier_claims where claim_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.carrier_claims where clm_grp_id BETWEEN ‘XXX’ AND ’YYY’;
+
+
+        -- For DME and Carrier
+        Select count(*) from public.carrier_claims where carr_clm_cntl_num in (‘XXX’,’YYY’,…);
+
+
+        -- For HHA, Hospice, Inpatient, Outpatient, SNF
+        Select count(*) from public.hha_claims where fi_doc_clm_cntl_num in (‘XXX’,’YYY’,…);
+
+
+
+2. Load data. Begin with TEST (Approval required for SBX and PROD)
+3. Create dated Pipeline folder (same format as CCW) in the 'Incoming' folder of the environments ETL bucket. 
+4. Upload generated synthetic data to dated folder.  Watch for Data Discovered and Data Loaded slack messages. Note: Also watch logs (at first)
+5. Repeat for each approved environment. 
+6. Upload synthetic data to shared space (TBD) example: test-synthetic-data S3 bucket (public)
+
+## Checking for collisions in TEST, SBX or PROD
+
+1. Run queries to find any possible duplicate parameters
```

Comment:
Can you mention here that these should be run as a sanity check after loading the data in each environment and that we expect them to return zero rows.